src/main/java/com/flightstats/hub/app/HubBindings.java

```diff
@@ -220,9 +258,11 @@ protected void configure() {
         bind(FinalCheck.class).to(SpokeFinalCheck.class).asEagerSingleton();
         bind(InFlightService.class).asEagerSingleton();
         bind(ChannelService.class).asEagerSingleton();
-        bind(MetricsConfig.class).toInstance(metricsConfig);
-        bind(MetricRegistry.class).toInstance(metricRegistry);
-        bind(JVMMetricsService.class).asEagerSingleton();
+        bind(MetricsConfig.class)
```

Comment:
this maaaay auto-bind as needed, but I'm not sure about that.

Reply:
If I get your drift, it should. I added this in order to use the annotation with the assumption that at a later date we could reuse this metrics config for our other metrics reporters. Would it be cleaner to make that change when we get to that point?